front/app/containers/Admin/formBuilder/containers/projects/forms/index.tsx

```diff
@@ -4,95 +4,61 @@ import { injectIntl } from 'utils/cl-intl';
   const project = useProject({ projectId });
+  const phases = usePhases(projectId);
+
+  if (isNilOrError(project)) {
+    return null;
+  }
 
-  const toggleSubmissionsEnabled = () => {
-    if (!isNilOrError(project)) {
-      updateProject(projectId, {
-        posting_enabled: !project?.attributes.posting_enabled,
-      });
-    }
-  };
+  const formActionsConfigs = getFormActionsConfig(project, phases);
 
   return (
-    <Box width="100%">
-      <Box display="flex" flexDirection="row" width="100%" mb="48px">
+    <>
+      <Box width="100%">
         <Box width="100%">
           <Title>{formatMessage(messages.survey)}</Title>
           <Text>{formatMessage(messages.surveyDescription)}</Text>
         </Box>
-        <Box
-          width="100%"
-          display="flex"
-          alignItems="center"
-          justifyContent="center"
-        >
-          {project?.attributes.posting_enabled !== undefined && (
-            <Toggle
-              checked={project?.attributes.posting_enabled}
-              label={formatMessage(messages.openForSubmissions)}
-              onChange={() => {
-                toggleSubmissionsEnabled();
-              }}
-            />
-          )}
-        </Box>
-      </Box>
-      <Box
-        display="flex"
-        flexDirection="row"
-        width="100%"
-        justifyContent="space-between"
-      >
-        <Button
-          icon="download"
-          buttonStyle="primary"
-          width="auto"
-          minWidth="312px"
-        >
-          {formatMessage(messages.downloadSurveyResults, {
-            count: 956, // TODO: Get this from the API
-          })}
-        </Button>
-        <Button
-          icon="edit"
-          buttonStyle="primary"
-          width="auto"
-          minWidth="312px"
-          onClick={() => {
-            clHistory.push(`/admin/projects/${projectId}/native-survey/edit`);
-          }}
-        >
-          {formatMessage(messages.editSurveyContent)}
-        </Button>
-        <Button
-          linkTo={`/projects/${project?.attributes.slug}/ideas/new`}
-          icon="eye"
-          openLinkInNewTab
-          buttonStyle="primary"
-          width="auto"
-          minWidth="312px"
-        >
-          {formatMessage(messages.viewSurveyText)}
-        </Button>
+        {formActionsConfigs.map(
+          (
+            { editFormLink, heading, postingEnabled, togglePostingEnabled },
+            index
+          ) => {
+            return (
+              <>
+                <FormActions
+                  editFormLink={editFormLink}
+                  viewFormLink={`/projects/${project?.attributes.slug}/ideas/new`}
+                  heading={heading}
+                  postingEnabled={postingEnabled}
+                  togglePostingEnabled={togglePostingEnabled}
+                />
+                {index !== formActionsConfigs.length - 1 && (
+                  <Box height="1px" border="1px solid #E0E0E0" />
```

Comment:
Do we have a named color we could use here instead?

Reply:
Nice catch, we actually do. Let me make the fix